src/sonic-config-engine/minigraph.py

```diff
@@ -191,6 +192,7 @@ def parse_dpg(dpg, hname):
             for addrtuple in pc_map.get(pcintfname, []):
                 pc_attributes.update(addrtuple)
                 pc_intfs.append(copy.deepcopy(pc_attributes))
+            pcs[pcintfname] = pc_attributes
```

Comment:
i would vote for removing the duplicated information but not here in this commit. i'll submit a separate pull request to address this as well as adding some unit tests to make sure that the generated configuration files are not changed before/after new code changes.